ui/DesignSystem/Component/Registration/NavigationButtons.svelte

```diff
@@ -0,0 +1,36 @@
+<script>
```

Comment:
Can we just name it `Navigation` or do we have another thing that is a navigation?

Reply:
I feel like `Navigation` is too vague. Also there are other ways we navigate through the app and this is specific to the registration flow so maybe it can be `RegistrationNavigation`?